### PR TITLE
feat(api-gen): generate separate files for CLI subcommands

### PR DESCRIPTION
**Pull request by JeanMeche**

### CODE REVIEW

File: bazel/api-gen/rendering/templates/cli-reference.tsx [hunk 1]
```diff
@@ -73,3 +53,12 @@ export function CliCommandReference(entry: CliCommandRenderable) {
     </div>
   );
 }
+
+
+function commandName(entry: CliCommandRenderable, command: string) {
+ if(entry.parentCommand?.name) {
```
Comment (josephperrott) [inline bazel/api-gen/rendering/templates/cli-reference.tsx:59]: Looks like this is missing a space?
```suggestion
 if (entry.parentCommand?.name) {
```

**Review by josephperrott — APPROVED** [review]
LGTM

File: bazel/api-gen/rendering/templates/cli-reference.tsx (hunk 1 above)
Comment (JeanMeche) [reply bazel/api-gen/rendering/templates/cli-reference.tsx]: ✅

**Review by twerske — APPROVED** [review]

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit edb9306719a545fd2803acacadeaea503c8218eb.

**Pull request closed without merging**